.github/workflows/cd.yml

```diff
@@ -115,15 +115,42 @@ jobs:
         run: |
           npx lerna version --no-private --conventional-commits --conventional-prerelease --preid=rc --no-changelog --yes
 
+      - name: version stable npm packages to npmjs.org
+        if: ${{ github.event_name == 'workflow_dispatch' && github.ref == 'refs/heads/main' && github.event.inputs.preid == 'stable' }}
+        run: |
+          npx lerna version --no-private --conventional-commits --conventional-graduate --no-changelog --yes
+
+      - name: update cli ai key
```

Comment:
the two steps share the same condition cannot be written in one step?

Reply:
this step use action can only modify one file once. or rewrite this action for two files' field change.